rmgpy/cantherm/molepro.py

```diff
@@ -28,10 +28,13 @@
 #                                                                             #
 ###############################################################################
 
+import math
+import numpy
 import os.path
-
 import rmgpy.constants as constants
 
+from rmgpy.statmech import IdealGasTranslation, NonlinearRotor, LinearRotor, HarmonicOscillator, Conformer
+
 class MoleProLog:
```

Comment:
In statmech.py we import `from rmgpy.cantherm.molpro import MolproLog`, but here the class name is `MoleProLog`. Could you fix this and make sure we only use the `Molpro` naming convention throughout this file?